### DISCUSSION TITLE
powerlevel10k theme inside cmder

### DISCUSSION THREAD

**Discussion by abmmhasan** [cmderdev/cmder#2613] [General]
Wanted to add powerlevel10k (zsh theme) into cmder. But couldn't. Can we have have a theme folder system where qew can put in such themes then can activate?

**Comment by daxgames** [comment]
I don't understand the request.  The `%cmder_root%\[config|bin|opt]` folders are yours to add anything you want to.

Can you explain what you tried to do?  Provide details and links.

**Reply by abmmhasan** [reply]
I wanna activate powerlevel10k (https://github.com/romkatv/powerlevel10k) theme in cmder.

**Reply by Eastegg** [reply]
Oh man, looks you gotta long way to go

**Reply by Eastegg** [reply]
powerlevel10k is for zsh , zsh you can install in msys2 or wsl.
